source/BatchDrawList.h

```diff
@@ -29,6 +29,17 @@ class Sprite;
 // This class collects a set of OpenGL draw commands to issue and groups them by
 // sprite, so all instances of each sprite can be drawn with a single command.
 class BatchDrawList {
+private:
```

Comment:
Private stuff goes after public and protected stuff.

Reply:
Not for structs

Reply:
Source?

Reply:
I note that the Conversation class declares some private nested classes (which we can consider equivalent to structs for this purpose) before its public members. This seems to have been done when adding conditions to conversation text nodes.
Those shouldn't be there, a person looking at a class definition is probably looking at it from the perspective of working on another class, so they don't need (or even want) to see the private members, certainly not before hte public members (since those are the ones they can actually use from other classes.)
We also shouldn't split members with the same access modifier, it's just confusing to see some private members and then public members and then more private members.
I'm moving the definitions in the Conversation class to a more appropriate location.

Reply:
After doing some research it seems like you are right I will change this

Reply:
done